controllers/authorization/authorization_controller.go

```diff
@@ -20,15 +21,20 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const ctrlName = "authorization"
```

Comment:
```suggestion
const ctrlName = "platform-authorization"
```

nit to be consistent with "platform-routing"

Reply:
It should actually be the other way around. The root `ctrlLog` is prefixed with `controllers.platform`

Reply:
Fixed in fc3e4b8